plugins/util.js

```diff
@@ -236,6 +287,99 @@ function generateIndexSearch({ posts }) {
   return indexJson;
 }
 
+/**
+ * getSitemapData
+ */
+
+function generateSitemap({ posts = [], pages = [] }) {
+  const { homepage = '' } = config;
+
+  const sitemap = `<?xml version="1.0" encoding="UTF-8"?>
+    <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+      <url>
+        <loc>${homepage}</loc>
+        <lastmod>${new Date().toISOString()}</lastmod>
+      </url>
+        ${pages
+          .map((slug) => {
+            return `<url>
+                      <loc>${homepage}/${slug}</loc>
+                      <priority>0.3</priority>
+                    </url>
+                `;
+          })
+          .join('')}
+          ${posts
+            .map((post) => {
+              return `<url>
+                        <loc>${homepage}/posts/${post.slug}</loc>
+                        <lastmod>${new Date(post.modified).toISOString()}</lastmod>
+                      </url>
+                  `;
+            })
+            .join('')}
+    </urlset>
+    `;
+
+  const sitemapFormatted = prettier.format(sitemap, {
```

Comment:
this is neat, didnt know you could do that

Reply:
Yeah! I discovered these days too. In fact, looking at it now, realized is not neccesary to pass the entire prettier configuration of the project, only with the HTML parser and printWidth will just work fine (as we don't need javascript or scss rules). I'll change it.